peps/pep-0772.rst

```diff
@@ -0,0 +1,463 @@
+PEP: 772
+Title: Packaging governance process
+Author: Barry Warsaw <barry@python.org>,
+        Deb Nicholson <deb@python.org>,
+        Pradyun Gedam <pradyunsg@gmail.com>
+Status: Draft
+Type: Process
+Topic: Governance, Packaging
+Created: 21-Jan-2025
+Replaces: 609
```

Comment:
PEP editors: should we add `Superseded-By: 772` to PEP 609, or only when this one is accepted?

Reply:
(not an editor, but author on both)

I'd prefer that we only add it once this PEP is accepted.